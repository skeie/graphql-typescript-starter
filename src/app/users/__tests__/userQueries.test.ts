import gql from 'graphql-tag';
import createServer from '../../../../test/support/createServer';
import { initDb, cleanDb } from '../../../../test/support/dbHelpers';

const USER_QUERY = gql`
  query UserQuery {
    user {
      email
    }
  }
`;

describe('userQueries', () => {
  let query;

  beforeAll(async () => {
    await initDb();
    const serverRes = await createServer();
    query = serverRes.query;
  });

  afterAll(async () => {
    await cleanDb();
  });

  it('should query user', async () => {
    const { data } = await query({
      query: USER_QUERY,
    });
    expect(data).toMatchSnapshot();
  });
});
