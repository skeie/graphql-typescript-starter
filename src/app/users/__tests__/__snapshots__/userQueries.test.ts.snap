// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`userQueries should query user 1`] = `
Object {
  "user": Object {
    "email": "mptestUser@mail.com",
  },
}
`;
